lib/fpm/package/dir.rb

```diff
@@ -203,27 +203,5 @@ def copy(source, destination)
     copy_metadata(source, destination)
   end # def copy
 
-  def copy_metadata(source, destination)
```

Comment:
I mentioned this in the main notes. I needed this function in the pacman class, so I put it in `FPM::Util`.